cucumber/api/features/support/rest_helpers.rb

```diff
@@ -11,7 +11,7 @@ def headers
     @headers ||= {}
   end
 
-  def post_json path, body, options = {}
+  def post_json(path, body, options = {})
```

Comment:
Thanks for adding parentheses around these arguments! I feel it makes the the argument intention much more clear.